src/explorer/model/Dependencies.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { ITreeItem } from "./ITreeItem";
+
+const DUPLICATE_VALUE: string = "omitted for duplicate";
+const CONFLICT_VALUE: string = "omitted for conflict";
+
+export class Dependencies implements ITreeItem {
+    public dependency: string;
+    public pomPath: string;
+    private curDep: string;
```

Comment:
does curDep stand for "current dependency"? What's the difference with another member `dependency`?

Reply:
better not to use abbr. in name of member